src/components/Avatar/Avatar.scss

```diff
@@ -1,9 +1,9 @@
 @import '../../styles/common';
 
 .Avatar {
-  --pc-avatar-extra-small-size: var(--p-space-6);
-  --pc-avatar-small-size: var(--p-space-8);
-  --pc-avatar-medium-size: var(--p-space-10);
+  --pc-avatar-extra-small-size: 24px;
```

Comment:
I'm not sure these size should be coupled with the spacing scale. If we made adjustments to the spacing scale, would we also expect avatar size to change?

Reply:
I don't think so, that's why I decoupled them. Might make our coverage worse but I think they should just be hard coded 👍

Reply:
Ha my bad I was reading the diff backwards! But yes agree! 👍